modules/virt-hot-plugging-a-virtual-disk-using-the-web-console.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-hot-plugging-virtual-disks.adoc
+
+[id="virt-hot-plugging-a-virtual-disk-using-the-web-console{context}"]
+= Hot-plugging a virtual disk using the web console
+
+Hot-plug virtual disks that you want to attach to a virtual machine instance (VMI) while a virtual machine is running.
+
+.Prerequisites
+* You must have a running virtual machine to hot-plug a virtual disk.
+
+.Procedure
+
+. Click *Workloads* → *Virtualization* from the side menu.
+
+. On the *Virtual Machines* tab, select the running virtual machine with the disk that you want to hot-plug.
```

Comment:
There is no disk attached to the VM at this point, so this sentence should be:

. On the *Virtual Machines* tab, select the running virtual machine to which you want to hot-plug a virtual disk.